src/main/java/org/spongepowered/common/mixin/core/server/MixinNetHandlerPlayServer.java

```diff
@@ -187,4 +193,32 @@ public void processCommandBlock(C17PacketCustomPayload packetIn, CallbackInfo ci
 
     }
 
+    @Inject(method = "processPlayer", at = @At(value = "FIELD", target = "net/minecraft/network/NetHandlerPlayServer.hasMoved", ordinal = 2), cancellable = true, remap = false)
```

Comment:
Oh. But why does the mixins library fails to find that [field](https://travis-ci.org/SpongePowered/SpongeCommon/jobs/61914584#L468)? 
I checked it and HasMoved isn't a [forge](https://github.com/MinecraftForge/MinecraftForge/blob/f0d96b7c50cb1aceb101c5e9c0d78806b87e1240/patches/minecraft/net/minecraft/network/NetHandlerPlayServer.java.patch) thingy :worried:.